dDatabase/CDClientDatabase/CDClientTables/CDMissionsTable.cpp

```diff
@@ -79,7 +79,7 @@ void CDMissionsTable::LoadValuesFromDatabase() {
 		entries.push_back(entry);
 		tableData.nextRow();
 	}
-
+	std::ranges::sort(entries, [](const CDMissions& lhs, const CDMissions& rhs) { return lhs.id < rhs.id; });
```

Comment:
Yknow, CDClient is sorted by ID if we were to start mmapping it.

Reply:
was left over from a previous thing we were doing to speed up the searches.  Could be removed now mr aron